src/com/google/javascript/jscomp/DefaultPassConfig.java

```diff
@@ -184,6 +185,38 @@ void maybeInitializeModuleRewriteState() {
     }
   }
 
+  private char[] getReservedNamingFirstChars(
```

Comment:
Done. I added an integration test for Angular. I can't add one for Polymer yet as it requires either #2277 or #2238 for the `polymerVersion` to be valid > 1.

If this lands first, I'll add an integration test to one of those PRs.